### DISCUSSION TITLE
Error Importing module from powershell gallery

### DISCUSSION THREAD

**Discussion by psinha4569** [thycotic-ps/thycotic.secretserver#302] [Q&A : Module] [answered]
I'm getting an error when I do a ` Import-Module -Name Thycotic.SecretServer`. It says it cannot load Thycotic.SecretServer.dll or one of its dependencies. How do I resolve this error?

Import-Module : Could not load file or assembly 'file:///C:\Program 
Files\WindowsPowerShell\Modules\thycotic.secretserver\bin\Thycotic.SecretServer.dll' or one of its dependencies. Operation is not 
supported. (Exception from HRESULT: 0x80131515)
At line:1 char:1
+ Import-Module -Name thycotic.secretserver
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Import-Module], FileLoadException
    + FullyQualifiedErrorId : System.IO.FileLoadException,Microsoft.PowerShell.Commands.ImportModuleCommand

**Comment by psinha4569** [comment, marked as answer]
I figured it out, Unblock-File before extracting it to modules directory

**Reply by jagger** [reply]
Glad to hear
